index.d.ts

```diff
@@ -2584,6 +2589,7 @@ declare namespace Eris {
     addRelationship(block?: boolean): Promise<void>;
     deleteNote(): Promise<void>;
     dynamicAvatarURL(format?: ImageFormat, size?: number): string;
+    dynamicBannerURL(format?: ImageFormat, size?: number): string;
```

Comment:
```suggestion
    dynamicBannerURL(format?: ImageFormat, size?: number): string | null;
```
Can also return `null`

Reply:
See [this comment](https://github.com/abalabahaha/eris/pull/1238#discussion_r663509170), I did the same thing for this file. 👍🏻